boards/default/distros/br/br.py

```diff
@@ -188,6 +188,22 @@ def configure(self, env):
         mergeScript = br_dir / 'merge_config.sh'
         wlutil.run([mergeScript] + kFrags, cwd=(br_dir / 'buildroot'), env=env)
 
+        # Fakeroot needs to be set to 1.31 to compile with sysroot 2.17
```

Comment:
So this needs to be updated everytime the sysroot changes?

Reply:
The bug is that Conda's sysroot 2.17 contains a header file against which the host fakeroot utility at BR's version (1.34) does not compile. This workaround force downgrade's buildroot's fakeroot to an older version that does compile with 2.17. AFAICT there is no regression from this, the update to 1.34 in buildroot seems to be just routine dependency maintenance.

If conda provided a sysroot newer than 2.17 (it currently does not) and we bumped to that, then this entire override thing can be removed.